File: Java/Product.java
public class Product {
    private String price;
    private String idProduct;

    public Product() {

    }

    public Product(String price, String idProduct) {
        this.price = price;
        this.idProduct = idProduct;
    }
    
    public void setPrice(String price) {
        this.price = price;
    }

    public String getPrice() {
        return this.price;
    }
    
    public void setIdProduct(String idProduct) {
        this.idProduct = idProduct;
    }
    
    public String getIdProduct() {
        return this.idProduct;
    }



}

File: Python/Product.py
class Product:
    __idProduct = ""
    __price = ""

    def __init__(self, idProduct = "", price = ""):
        self.__idProduct = idProduct
        self.__price = price

    def setIdProduct(self, idProduct):
        self.__idProduct = idProduct
    
    def getIdProduct(self):
        return self.__idProduct

    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price
    